cmd/earthly/subcmd/build_cmd.go

```diff
@@ -923,7 +923,13 @@ func (a *Build) logShareLink(ctx context.Context, cloudClient *cloud.Client, tar
 	return logstreamURL, true, printLinkFn
 }
 
-func (a *Build) maybePrintBuildMinutesInfo(orgName string) {
+func (a *Build) maybePrintBuildMinutesInfo() {
+	orgName := a.cli.OrgName()
+	settings := a.cli.Flags().BuildkitdSettings
+	if settings.SatelliteName == "" || settings.SatelliteIsSelfHosted {
```

Comment:
can we use [IsUsingSatellite](https://github.com/earthly/earthly/blob/2928bb846c10bb86aad04fcd0003d89782072690/cmd/earthly/subcmd/cli.go#L35) to make is more consistent?